integration_tests/integration_dotnet_test.go

```diff
@@ -535,42 +545,127 @@ func TestDeletedWith(t *testing.T) {
 func TestProviderCall(t *testing.T) {
 	const testDir = "provider_call"
 	testDotnetProgram(t, &integration.ProgramTestOptions{
-		Dir:            filepath.Join(testDir, "dotnet"),
-        Env:            []string{"TEST_VALUE=HelloWorld"},
-		Quick:          true,
+		Dir:   filepath.Join(testDir, "dotnet"),
+		Env:   []string{"TEST_VALUE=HelloWorld"},
+		Quick: true,
 	})
 }
 
 func TestProviderCallInvalidArgument(t *testing.T) {
 	const testDir = "provider_call"
 	testDotnetProgram(t, &integration.ProgramTestOptions{
-		Dir:            filepath.Join(testDir, "dotnet"),
-        Env:            []string{"TEST_VALUE="},
-        ExpectFailure:  true,
-		Quick:          true,
+		Dir:           filepath.Join(testDir, "dotnet"),
+		Env:           []string{"TEST_VALUE="},
+		ExpectFailure: true,
+		Quick:         true,
 	})
 }
 
 func TestProviderConstruct(t *testing.T) {
 	const testDir = "provider_construct"
 	testDotnetProgram(t, &integration.ProgramTestOptions{
-		Dir:            filepath.Join(testDir, "dotnet"),
-		Quick:          true,
+		Dir:   filepath.Join(testDir, "dotnet"),
+		Quick: true,
 	})
 }
 
 func TestProviderConstructDependencies(t *testing.T) {
 	const testDir = "provider_construct_dependencies"
 	testDotnetProgram(t, &integration.ProgramTestOptions{
-		Dir:            filepath.Join(testDir, "dotnet"),
-		Quick:          true,
+		Dir:   filepath.Join(testDir, "dotnet"),
+		Quick: true,
 	})
 }
 
 func TestProviderConstructUnknown(t *testing.T) {
 	const testDir = "provider_construct_unknown"
 	testDotnetProgram(t, &integration.ProgramTestOptions{
-		Dir:            filepath.Join(testDir, "dotnet"),
-		Quick:          true,
+		Dir:   filepath.Join(testDir, "dotnet"),
+		Quick: true,
 	})
 }
+
+func readUpdateEventLog(logfile string) ([]apitype.EngineEvent, error) {
+	events := make([]apitype.EngineEvent, 0)
+	eventsFile, err := os.Open(logfile)
+	if err != nil {
+		if os.IsNotExist(err) {
+			return nil, nil
+		}
+		return nil, fmt.Errorf("expected to be able to open event log file %s: %w",
+			logfile, err)
+	}
+
+	defer contract.IgnoreClose(eventsFile)
+
+	//	f, _ := ioutil.ReadAll(eventsFile)
+	//	fmt.Println("eventsfile", string(f))
```

Comment:
Do we need to keep these?

Reply:
Whoops, no, this was just for debugging and I forgot to remove it.